src/scss/3_components/form/_webform.scss

```diff
@@ -47,3 +47,32 @@
     margin-top: 0;
   }
 }
+
+.webform-submission-form input[type="submit"] {
+  background: $link none;
+  min-width: 13rem;
+  @include adjust-font-size-to(18px);
+  line-height: 1.5;
+  border: 3px solid transparent;
+  color: $white;
+  font-weight: normal;
+  text-shadow: none;
+  text-underline-offset: .25em;
+  -webkit-box-shadow: none;
```

Comment:
@David-Corbett Am I right in thinking that this is required as autoprefixer should generate the vendor specific styles?

Reply:
I think you are right, it doesn't need to be there.